sql/core/src/test/scala/org/apache/spark/sql/MathExpressionsSuite.scala

```diff
@@ -262,4 +263,10 @@ class MathExpressionsSuite extends QueryTest {
       ctx.sql("SELECT negative(1), negative(0), negative(-1)"),
       Row(-1, 0, 1))
   }
+
+  test("positive") {
+    checkAnswer(
+      ctx.sql("""SELECT positive(1), positive(-1), positive("abc")"""),
```

Comment:
Use `selectExpr` instead, we'd better keep the same pattern with the other test case in the same file.